src/main/java/com/future/function/qa/pages/base/MenuPage.java

```diff
@@ -15,6 +15,11 @@ public WebElementFacade comparisonsMenu(int menuIndex) {
     return find(By.xpath(String.format("//body//li[%s]", menuIndex)));
   }
 
+  public WebElementFacade gradesMenu(int menuIndex) {
+
+    return find(By.xpath(String.format("//body//li[%s]", menuIndex)));
```

Comment:
you can use karnando's code in HomePage where you can get the menu with its index

Reply:
Done